package.json

```diff
@@ -268,5 +268,8 @@
             "@modules/(.*)$": "<rootDir>/src/ducks/modules/$1",
             "^react-native$": "react-native-web"
         }
+    },
+    "dependencies": {
+        "node-fetch": "^1.7.3"
```

Comment:
Because we bundle our code using webpack, we don't use the `dependencies` section of the main package file. If the dependency is going to be bundled, add it to `devDependencies`, or alternatively if it is going to be used unbundled at runtime, add it to `public/package.json`.